packages/stark-core/src/modules/error-handler.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./error-handler/actions";
+export * from "./error-handler/error-handling";
```

Comment:
This folder containing the error handler class should be called `handlers` (as it was in your fist commit) instead of `error-handling` :-)